cli/src/commands/edit.rs

```diff
@@ -21,10 +21,12 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Loads the contents of the specified revision and sets it as the active changeset.
```

Comment:
nit: I think we quite consistently don't include a trailing period on the first line (the part that shows up in `jj -h`)

Reply:
I did notice that, I kept accidentally typing it :sweat_smile: Failed to catch it the last time I guess